docs/src/examples/components/Text/Types/TextSizesExample.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Text } from '@stardust-ui/react'
+
+const TextSizesExample = () => (
+  <div>
+    <Text size="xs">Dicta voluptatum dolorem.</Text>
```

Comment:
actually, this raises another point related to the values used for sizes. The problem is that it is hard to intuitively reason for the client about, say, which one is bigger: `xl` or `2x`. Would propose to use values that are more intuitive in this sense (and address the issue by means of separate PR)

Reply:
Completely agree. My next PR is going to use a theme-based enum to show the appropriate sizes. 

What I'm still not sure about is what to do for naming and how that could vary among themes...